waffle_hub/schema/result.py

```diff
@@ -23,5 +23,10 @@ class InferenceResult(BaseSchema):
 
 
 @dataclass
-class ExportResult(BaseSchema):
-    export_file: str = None
+class ExportOnnxResult(BaseSchema):
+    export_onnx_file: str = None
+
+
+@dataclass
+class ExportWaffleResult(BaseSchema):
+    export_waffle_file: str = None
```

Comment:
I think the naming `export_file` I named is wierd 😅.

`export_onnx_file` -> `onnx_file`
`export_waffle_file` -> `waffle_file`
would be better.

@Jiyong-Jeon

Reply:
I think so, too.

However, I think all variables should be changed by clearly setting the variable naming convention as 'onnx_file' -> 'onnx_file_path' or 'onnx_file_name'.

So if the variable is a name, I'll add '_name' and if it's a path, I'll add '_path' to ensure the naming.

Do you agree? @ZeroAct

Reply:
This results in a lot of changes. 
Please predict the side effects and notify to everyone.

Reply:
When I modified it to match the naming convention, I found too many changes on the user interface.

- Dataset
  - property
    - `dataset_info_file` -> `dataset_info_file_path`
    - `train_set_file` -> `train_set_file_path`
    - `val_set_file` -> `val_set_file_path`
    - `test_set_file` -> `test_set_file_path`
    - `unlabeled_set_file` -> `unlabeled_set_file_path`
  - Classmethod
    - from_coco
      - `coco_file` -> `coco_file_path`
      - `coco_files` -> `coco_file_paths`
    - from_autocare_dlt
      - `coco_file` -> `coco_file_path`
    - from_label_studio
      - `json_file` -> `json_file_path`

- result schema
  - TrainResult
    - `best_ckpt_file` -> `best_ckpt_file_path`
    - `last_ckpt_file` -> `last_ckpt_file_path`
  - ExportOnnxResult
    - `export_onnx_file` -> `export_onnx_file_path`
  - ExportWaffleResult
    - `export_waffle_file` -> `export_waffle_file_path`

- Hub
  - property
    - `MODEL_CONFIG_FILE` -> `MODEL_CONFIG_FILE_NAME`
    - `TRAIN_CONFIG_FILE` -> `TRAIN_CONFIG_FILE_NAME`
    - `BEST_CKPT_FILE` -> `BEST_CKPT_FILE_NAME`
    - `LAST_CKPT_FILE` -> `LAST_CKPT_FILE_NAME`
    - `METRIC_FILE` -> `METRIC_FILE_NAME`
    - `EVALUATE_FILE` -> `EVALUATE_FILE_NAME`
    - `INFERENCE_FILE` -> `INFERENCE_FILE_NAME`
    - `ONNX_FILE` -> `ONNX_FILE_NAME`
    - `model_config_file` -> `model_config_file_path`
    - `inference_file` -> `inference_file_path`
    - `train_config_file` -> `train_config_file_path`
    - `onnx_file` -> `onnx_file_path`
    - `evaluate_file` -> `evaluate_file_path`
    - `best_ckpt_file` -> `best_ckpt_file_path`
    - `last_ckpt_file` -> `last_ckpt_file_path`
  - Classmethod
    - from_model_config (args)
      - `model_config_file` -> `model_config_file_path`
    - from_waffle_file (args)
      - `waffle_file` -> `waffle_file_path`
  - CustomCallback args (transformers_hubs.py)
    - `metric_file` -> `metric_file_path`
  
- Labeled Dataset (utils)
  - `set_file` -> `set_file_path`
 
- etc..

I think it's too critical for other apps using "waffle_hub", so I'll discuss it more closely in the next sprint and change it.